src/components/CloudBrute/CloudBrute.tsx

```diff
@@ -0,0 +1,173 @@
+            } else {
+                handleProcessData(`\nProcess terminated with exit code: ${code} and signal code: ${signal}`);
+            }
+
+            setPid(""); // Clear the child process pid reference.
+            setLoading(false); // Cancel the loading overlay.
+        },
+        [handleProcessData] // Dependency on the handleProcessData callback
+    );
+
+    /**
+     * onSubmit: Asynchronous handler for the form submission event.
+     * It sets up and triggers the CloudBrute tool with the given parameters.
+     * Once the command is executed, the results or errors are displayed in the output.
+     *
+     * @param {FormValuesType} values - The form values, containing the domain, keyword, and wordlist path.
+     */
+    const onSubmit = async (values: FormValuesType) => {
+        setLoading(true);
+        const args = ["-d", values.domain, "-k", values.keyword, "-w", values.wordlist];
+        CommandHelper.runCommandGetPidAndOutput("cloudbrute", args, handleProcessData, handleProcessTermination)
+            .then(({ pid, output }) => {
+                setPid(pid);
+                setOutput(output);
+            })
+            .catch((error) => {
+                setLoading(false);
+                setOutput(`Error: ${error.message}`);
+            });
+    };
+
+    /**
+     * Clears the output state.
+     */
+    const clearOutput = useCallback(() => {
+        setOutput("");
+    }, [setOutput]);
+
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButton(loading, pid)}
+                    <TextInput
+                        label="Target Domain"
+                        required
+                        placeholder="e.g., google.com"
+                        {...form.getInputProps("domain")}
+                    />
+                    <TextInput label="Keyword" required placeholder="e.g., test" {...form.getInputProps("keyword")} />
+                    <TextInput
+                        label="Path to Wordlist"
+                        required
+                        placeholder="/usr/share/dirb/wordlists/common.txt"
+                        {...form.getInputProps("wordlist")}
+                    />
+                    {SaveOutputToTextFile(output)}
+                    <Button type="submit">Run CloudBrute</Button>
+                    <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
```

Comment:
You forgot to add {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
```suggestion
                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
                    <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
```